### ISSUE TITLE
Is 404.html ever displayed?

### ISSUE THREAD

**Issue by carlinmack** [WDscholia/scholia#1536]
404.html implies that it would be a page that looks like Scholia (with the top menu etc) with the content: "Not found". However I can only get the page which says:

> Not Found

> The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.

Which is the Flask error message.

Should we not be serving 404.html instead?

**Comment by egonw** [comment]
The 404.html shows up for redirects, as it did for `/doi/$DOI` and `/inchikey/$DOI`, tho these two particular examples now have dedicated 404 pages.

**Comment by carlinmack** [comment]
Yeah, I have since discovered that there are many URLs which show the current 404 page, like https://scholia.toolforge.org/location/Q5465/curation. I'll improve the above PR for the defined 404's
